Dockerfile

```diff
@@ -26,19 +26,19 @@ RUN mkdir -p $SCALA_HOME && cd $SCALA_HOME  \
     org.scala-lang scala-library $SCALA_FVER \
     # compiler
     org.scala-lang scala-compiler $SCALA_FVER \
-    org.scala-lang.modules scala-parser-combinators_$SCALA_VER 1.0.7 \
+    org.scala-lang.modules scala-parser-combinators_$SCALA_VER 1.1.2 \
  && ivy_install -n "grade-scala" -d "$SCALA_DIR/lib" \
     # These go to classpath
     # core libs are repeated, so the deps are resolved to same jars
     org.scala-lang scala-library $SCALA_FVER \
     org.scala-lang scala-compiler $SCALA_FVER \
     # extra libs
-    org.scala-lang.modules scala-swing_$SCALA_VER 2.0.3 \
+    org.scala-lang.modules scala-swing_$SCALA_VER 2.1.1 \
     # for grading
-    org.scalatest scalatest_$SCALA_VER 3.0.5 "default->master,compile,runtime" \
-    org.scalamock scalamock_$SCALA_VER [4.1.0,4.2[ \
-    org.scalastyle scalastyle_$SCALA_VER [1.0.0,1.1[ \
-    com.typesafe.akka akka-actor_$SCALA_VER [2.5.20,2.6[ \
+    org.scalatest scalatest_$SCALA_VER 3.1.1 "default->master,compile,runtime" \
+    org.scalamock scalamock_$SCALA_VER [4.3.0,4.4[ \
+    com.beautiful-scala scalastyle_$SCALA_VER [1.4.0,1.5[ \
```

Comment:
Is  `com.beautiful-scala` direct upgrade from `org.scalastyle`? Does it require configuration changes? If so, can the scalastyle be installed alongside beautiful-scala?